src/test/py/bazel/options_test.py

```diff
@@ -269,6 +269,18 @@ def testCommonPseudoCommand_unsupportedOptionValue(self):
         stderr,
     )
 
+  def testCommonPseudoCommand_version(self):
```

Comment:
If what you're testing is specific to starlark flags, include that in the test method name.